contrib/IECoreAppleseed/src/IECoreAppleseed/ParameterAlgo.cpp

```diff
@@ -145,49 +144,46 @@ string dataToString( const Data *value )
 	switch( value->typeId() )
 	{
 		case IntDataTypeId :
-			{
-				int x = static_cast<const IntData*>( value )->readable();
-				ss << x;
-			}
-			break;
+		{
+			int x = static_cast<const IntData*>( value )->readable();
+			ss << x;
+		}
+		break;
 
 		case FloatDataTypeId :
-			{
-				float x = static_cast<const FloatData*>( value )->readable();
-				ss << x;
-			}
-			break;
+		{
+			float x = static_cast<const FloatData*>( value )->readable();
+			ss << x;
+		}
+		break;
 
 		case StringDataTypeId :
-			{
-				const string &x = static_cast<const StringData*>( value )->readable();
-				ss << x;
-			}
-			break;
+		{
+			const string &x = static_cast<const StringData*>( value )->readable();
+			ss << x;
+		}
+		break;
 
 		case V2iDataTypeId :
-			{
-				const V2i &x = static_cast<const V2iData*>( value )->readable();
-				ss << x.x << ", " << x.y;
-			}
-			break;
+		{
+			const V2i &x = static_cast<const V2iData*>( value )->readable();
+			ss << x.x << ", " << x.y;
+		}
+		break;
 
 		case Color3fDataTypeId :
-			{
-				const Color3f &x = static_cast<const Color3fData*>( value )->readable();
-				ss << x.x << ", " << x.y << ", " << x.z;
-			}
-			break;
-		case BoolDataTypeId :
-			{
-				bool x = static_cast<const BoolData*>( value )->readable();
-				ss << x;
-			}
-			break;
+		{
+			const Color3f &x = static_cast<const Color3fData*>( value )->readable();
+			ss << x.x << ", " << x.y << ", " << x.z;
+		}
+		break;
 
-		default:
```

Comment:
Removing these defaults is causing compilation errors like the following :

```
contrib/IECoreAppleseed/src/IECoreAppleseed/ParameterAlgo.cpp:144:8: error: enumeration value 'LastExtensionTypeId' not handled in switch [-Werror=switch]
```

Perhaps we should be keeping the messages anyway, otherwise we have no indication of when something might not be having the expected effect?

Reply:
The messages were making the InteractiveRenderTest fail. That's why I removed them.